my-app/src/todoApp.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import Add from './components/add.js'
+import Search from './components/search.js'
+
+class TodoApp extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+      searchitems : []
+    };
+  }
+
+  deleteElement(itemIndex) {
+    this.state.items.splice(itemIndex, 1);
+    this.setState({items : this.state.items});
+  }
+
+  editElement(key) {
+    var newList = Object.assign([],this.state.items);
+    var newValue =  this._editValue.value;
+    newList.map( (value,e)=>{
```

Comment:
Give proper names to a variable, You are referring 'e' as key here. Generally, we used 'e' for event